src/main/groovy/org/jenkinsci/plugins/pipeline/modeldefinition/model/Agent.groovy

```diff
@@ -46,6 +46,9 @@ public class Agent implements Serializable {
     String label
 
     @Whitelisted
+    String args
```

Comment:
So this whole thing is going to get rewritten in the fairly near future, but I see no reason to reject this change in the meantime - it's a good addition that'll translate well into the pluggable approach we'll be moving towards. Buuuuut - could you go with `dockerArgs` instead of just `args` so that we don't end up with confusion down the road with combinations of keys, etc?